### PR TITLE
Add mirrord vs TP blog post

### PR DESCRIPTION
**Pull request by eyalb181**

### CODE REVIEW

File: content/en/blog/mirrord-vs-telepresence/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+5. As opposed to Telepresence which works at the network level, mirrord works at the pod level. This means it also **supports pods that aren’t exposed through a service**, and gives your local code access to anything the pod can access, including **components outside the cluster**. It also natively supports environments with service mesh like Linkerd or Istio.
+6. Because the mirrord client works at the process level, you can easily **configure exactly what’s executed remotely and what stays local**. For example, you could make all the components of the remote cluster accessible to your local process, but have it write to the database locally; or have it read files from the local file system, except a few specific files that you want to get from the remote pod. 
+7. mirrord is a tool for developers, and as such, in addition to a CLI tool, it also comes as an **IDE plugin for either VS Code or the IntelliJ IDEs**. This is the most popular way to use mirrord, where it naturally fits into the existing development workflow.
```
Comment (meowjesty) [inline content/en/blog/mirrord-vs-telepresence/index.md:34]:
```suggestion
7. mirrord is a tool for developers, and as such, in addition to a CLI tool, it also comes as an **IDE plugin for both VS Code and the IntelliJ IDEs**. This is the most popular way to use mirrord, where it naturally fits into the existing development workflow.
```

File: content/en/blog/mirrord-vs-telepresence/cli-command.png
Comment (meowjesty) [inline content/en/blog/mirrord-vs-telepresence/cli-command.png:1]: Image is blurry on the site!

**Review by meowjesty — CHANGES_REQUESTED** [review]
Out of focus image?

Can we turn this into svg / text?

**Review by meowjesty — APPROVED** [review]
🐻 👍

**Review by infiniteregrets — CHANGES_REQUESTED** [review]
content/en/blog/mirrord-vs-telepresence/mirrord-vs-telepresence.png is also blurry

**Comment by eyalb181** [comment]
> content/en/blog/mirrord-vs-telepresence/mirrord-vs-telepresence.png is also blurry

@infiniteregrets Try now

**Review by infiniteregrets — APPROVED** [review]

**Pull request merged**